README.md

```diff
@@ -1,10 +1,10 @@
-# SF-X Developer Tools
-This repository contains various scripts that support the distribution and development of the [SF-X Logic](https://github.com/groupby/sfx-logic) and [SF-X View](https://github.com/groupby/sfx-view) repositories.
+# GroupBy Elements Developer Tools
+This repository contains various scripts that support the distribution and development of the [GroupBy Elements Logic](https://github.com/groupby/elements-logic) and [GroupBy Elements View](https://github.com/groupby/elements-view) repositories.
 
 ## Scripts
 
 ### `setup.sh`
-This script installs dependencies and builds all the packages contained within an SF-X monorepo. This script assumes that a [yarn workspace](https://yarnpkg.com/lang/en/docs/workspaces/) is set up and that the root of the workspace is one directory above the directory containing this script.
+This script installs dependencies and builds all the packages contained within a GB Elements monorepo. This script assumes that a [yarn workspace](https://yarnpkg.com/lang/en/docs/workspaces/) is set up and that the root of the workspace is one directory above the directory containing this script.
 
 ### `build.sh`
-This script builds a single package. It is meant to be run at the root of an SF-X package. It is assumed that `typescript` is installed as a devDependency and this script is being run as an npm script.
+This script builds a single package. It is meant to be run at the root of an GB Elements package. It is assumed that `typescript` is installed as a devDependency and this script is being run as an npm script.
```

Comment:
should it be `a GB`? also we are introducing GroupBy Elements and GB Elements interchangeably?

Reply:
@ssbelle I suppose I am introducing that, yes.

The problem is that "GroupBy Elements" is a mouthful, and becomes very repetitive to see it in text. Additionally, Chantal and I have been trying to call it just Elements, but we'll have a problem doing that in the View Layer because the term 'elements' is overloaded. So we'll likely be using GB Elements in most cases there, with maybe only the occasional reference to 'Elements' if it's perfectly clear.

The following are therefore being used as equivalent:
* GroupBy Elements
* GB Elements
* Elements (very rarely)

There are cases where we're referring to the 'Elements-View' package as well. This is at least more specific and less likely to be confused with some other meaning of the word 'elements'.

Lastly, yes, it should have been 'a GB Elements package'. Thanks!